src/main/java/se/sundsvall/partyassets/service/StatusService.java

```diff
@@ -31,12 +31,12 @@ public StatusService(final StatusRepository repository) {
 	}
 
 	@Cacheable(value = CACHE_NAME, key = "{#root.methodName}")
```

Comment:
municipalityId needs to be added to the `@Cachable` (and `@CacheEvict`) annotation for the cache to work correctly. See for example how it is done for the status attribute.

Comment goes for all cacheable methods where municipalityId has been added as parameter.